modules/yugabytedb/options.go

```diff
@@ -0,0 +1,53 @@
+package yugabytedb
+
+import (
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// WithDatabaseName sets the initial database name for the yugabyteDB container.
+func WithDatabaseName(ysqlDBName string) testcontainers.CustomizeRequestOption {
```

Comment:
Thanks, I think that's much more readable now 👍

Reply:
You're welcome :) Happy to receive your feedbacks :)